lib/nimble_csv.ex

```diff
@@ -326,6 +335,55 @@ defmodule NimbleCSV do
         |> parse_enumerable(opts)
       end
 
+      def to_line_stream(stream) do
+        newline = :binary.compile_pattern(@encoded_newlines)
+
+        stream
+        |> Stream.chunk_while(
```

Comment:
We don't have this available on v1.3 but we do have it on v1.5. It is fine to bump the requirement to ~> 1.5. Just please remember to upgrade mix.exs and CI files. :)